Detectors/FIT/FT0/reconstruction/include/FT0Reconstruction/CTFCoder.h

```diff
@@ -141,28 +147,33 @@ void CTFCoder::decompress(const CompressedDigits& cd, VDIG& digitVec, VCHAN& cha
     firstEntry = channelVec.size();
     uint8_t chID = 0;
     int amplA = 0, amplC = 0, timeA = 0, timeC = 0;
+    int mTime_trg_gate = 192; // #channels
```

Comment:
If this a constant fixed forever, then it is better to define it in the constants file. If it is a parameter, it should come from external object (in future, loaded from the ccdb).